.config-schema.json

```diff
@@ -175,6 +175,12 @@
                     "type": "duration",
                     "default": "connections are not closed due to connection's age - database/sql default",
                     "x-env-variable": "OPENFGA_DATASTORE_CONN_MAX_LIFETIME"
+                },
+                "metricsEnabled": {
```

Comment:
One last thing here - could we nest this so that when/if we allow more selective metric collection it can be configured more similarly to the other configs we have. For example, we have:

`metrics.enabled`, `metrics.addr`, etc..

I can forsee a scenario where you only want to export a handful of the available metrics reported by a DB stats collector, and so having this as a nested field would be preferable in that regard. We may not always choose to use the default prometheus DBStatsCollector, and so this very well could have more configuration options.

So we'd have

`OPENFGA_DATASTORE_METRICS_ENABLED` and `datastore.metrics.enabled` for now, but in the future we may also have `datastore.metrics.include=["open_conns", "idle_conns"]` which would only track those metrics.

Reply:
@jon-whit done, would you please take a look? 🙏 I will update the documentation website / PR once this gets green light.